TramsDataApi/Migrations/TramsDb/TramsDbContextModelSnapshot.cs

```diff
@@ -961,6 +961,133 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.ToTable("AcademyTransferProjects","sdd");
                 });
 
+            modelBuilder.Entity("TramsDataApi.DatabaseModels.Concerns.Case.Management.Actions.Decisions.Decision", b =>
+                {
+                    b.Property<int>("DecisionId")
+                        .ValueGeneratedOnAdd()
+                        .HasColumnType("int")
+                        .HasAnnotation("SqlServer:ValueGenerationStrategy", SqlServerValueGenerationStrategy.IdentityColumn);
+
+                    b.Property<int>("ConcernsCaseId")
+                        .HasColumnType("int");
+
+                    b.Property<DateTimeOffset>("CreatedAtDateTimeOffset")
+                        .HasColumnType("datetimeoffset");
+
+                    b.Property<string>("CrmCaseNumber")
+                        .HasColumnType("nvarchar(max)");
```

Comment:
It would be good to start using appropriately sized fields to be kind to the database :)

Reply:
Can be updated when I have answers as to what the sizes are